Deploy/pack_minian_run.spec

```diff
@@ -22,6 +23,8 @@ datas += tmp_ret[0]; binaries += tmp_ret[1]; hiddenimports += tmp_ret[2]
 
 binaries += collect_dynamic_libs('llvmlite',destdir='.\\Library\\bin')
 
+excludes = ["IPython", "matplotlib", "PyQt5", "Markdown", "jupyter"]
```

Comment:
Nice! I hope it'll work!

Reply:
Can it be done for caiman as well?

Reply:
For the moment it work. Only for `jupyter` it don't really remove everything. It look like there are style some folder with almost nothing in them.

I think it can be done for caiman but I will add that when refactoring caiman

Reply:
oki